assets/home/tasks-section/ready.js

```diff
@@ -33,6 +34,22 @@ const Ready = ( { onDismiss } ) => {
 		} );
 	};
 
+	// Prepare social media links.
+	const shareLink = encodeURIComponent( coursePermalink );
+	const facebookLink = `https://www.facebook.com/sharer/sharer.php?u=${ shareLink }`;
+	const tumblerLink = sprintf(
```

Comment:
I think we should call it `tumblrLink` here, right?

```suggestion
	const tumblrLink = sprintf(
```

Reply:
Oops!